lib/https.js

```diff
@@ -165,6 +165,7 @@ function createConnection(port, host, options) {
   return socket;
 }
 
+// eslint-disable-next-line jsdoc/require-returns-check
```

Comment:
I think this one should be fixed. It should use `@constructor` rather than `@returns`. (The others should be fixed too but this one is really simple.)

```suggestion
```